contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -42,6 +42,10 @@ def _transmit(self, envelopes):
         This function should never throw exception.
         """
         try:
+            proxies = None
+            if self.options.proxy:
+                proxies = json.loads(self.options.proxy)
```

Comment:
```suggestion
                proxies = json.loads(self.options.proxies)
```

Reply:
I like the idea about making sure the variable name matches the options name. I had used `self.options.proxy` to be backward compatible with the existing ```proxy``` Option. Are we comfortable updating either renaming the existing ```proxy``` field or introducing a new ```proxies``` Option with the default value of None? 

Since the currently ```proxy``` variable isn't actually used in Master, to reduce duplication, I'm leaning towards the rename so wanted to get your thoughts on that.

Reply:
Agreed. Since `proxy` isn't implemented, the name change should be fine.